src/main/java/hudson/plugins/audit_trail/AuditTrailFilter.java

```diff
@@ -94,4 +111,27 @@ public void doFilter(ServletRequest request, ServletResponse res, FilterChain ch
 
     public void destroy() {
     }
+
+    // the default milestone doesn't seem right, as the injector is not available yet (at least with the JenkinsRule)
+    @Initializer(after = PLUGINS_PREPARED)
```

Comment:
Maybe it should be EXTENSIONS_AUGMENTED? I'm not sure when the descriptor lists are populated.

Reply:
I did this because I had issue (in the tests IIRC) with the injector not finding the class to inject. I took the first milestone that ensured class loading was ready. I suppose it wouldn't be an issue using `EXTENSIONS_AUGMENTED ` as the milestone is later in the lifecycle. Is there any advantage doing that? Or is this a standard?

Reply:
More than an advantage, I find it weird that the plugin uses an extension before the EXTENSIONS_AUGMENTED is reached. But if it works fine, I guess there's no real issue.

Reply:
Ok, I'll keep it as is for now as it's confirmed working. Thanks for the explanation!